docs/recipes/MaestroSetup.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Maestro Setup
+description: Setting up e2e testing with Maestro in Ignite
+tags:
+  - Maestro
+  - testing
+last_update:
+  author: Dan Edwards
+publish_date: 2023-01-06
+---
+
+# Setting Up Maestro in Ignite
+
+## Overview
+
+E2E testing is a critical part of any application but it can be difficult to set up and maintain. Maestro is a tool that promises to be easy to set up and maintain e2e tests. This recipe will walk you through setting up Maestro in your Ignite project.
+
+## Maestro Installation
+
+We're going to start by installing Maestro via the terminal. To do this, we'll need to run the following command:
```

Comment:
Let's incorporate a link to the Maestro docs here as well in the event someone needs to troubleshoot an installation